site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>Don't print all the predictions.</p><p><br></p><p>The first five predictions with their real labels should convey the idea.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
Done